collect_app/src/main/res/layout/form_entry_end.xml

```diff
@@ -76,30 +75,41 @@ the specific language governing permissions and limitations under the License.
             </androidx.constraintlayout.widget.ConstraintLayout>
         </com.google.android.material.card.MaterialCardView>
 
-        <com.google.android.material.button.MaterialButton
-            android:id="@+id/save_as_draft"
-            style="@style/Widget.Material3.Button.OutlinedButton"
+        <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
I think there must be a way to do this without nested `ConstraintLayout`s (maybe using `Barrier`), but it's really awkward to work out. Happy to go with this for now.